render_pdf/tex_files/tufte-handout.cls

```diff
@@ -0,0 +1,39 @@
+\NeedsTeXFormat{LaTeX2e}[1994/06/01]
```

Comment:
is it important to keep this name? Maybe we should just rename and remove references to Tufte

Reply:
I assume this comment is about the name of this file: I think this is an existing latex package called tufte-something, not code written by @skmnktl — ideally this should not be checked in and we should just use the one from TeX Live or CTAN or whatever, but such cleanup can happen when this PR is closer to being checked in :) (And I'm not sure we need this package in the first place)

Reply:
Sure, feel free to keep it for now… (Isn't it included as part of TeX Live or MikTeX or whatever distribution you're using, BTW?)

(What I meant by not needing the package is that, in principle, when we're generating things programmatically we don't necessarily need any LaTeX package because we could just output the corresponding TeX primitives directly, to get identical typeset output… but that's more of a theoretical concern as there aren't practical alternatives right now.)